lte/gateway/docker/docker-compose.yaml

```diff
@@ -359,3 +359,53 @@ services:
       timeout: "4s"
       retries: 3
     command: /var/opt/magma/bin/envoy_controller
+
+  # for docker compose v1: https://github.com/docker/compose/issues/8351#issuecomment-1085811366
+  # also see https://docs.docker.com/compose/compose-file/#depends_on
+  waitForHealthyPods:
+    image: alpine:latest
```

Comment:
Is the image actually needed here?

Reply:
Without an image, it would not be a valid definition afaik. [The documentation](https://docs.docker.com/compose/compose-file/build/#definitions) also reads like that for me.
It does not necessarily need to be an alpine, however, this is a very small and well known image, such that the overhead is minimal.